src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
The whitespace changes in this file will be re-generated by VS when we next edit resx.